display_information/add-graphics-with-renderer/src/main/java/com/esri/samples/add_graphics_with_renderer/AddGraphicsWithRendererSample.java

```diff
@@ -108,18 +118,83 @@ public void start(Stage stage) {
       polygonGeometry.addPoint(-20e5, -20e5);
       Graphic polygonGraphic = new Graphic(polygonGeometry.toGeometry());
       polygonGraphicOverlay.getGraphics().add(polygonGraphic);
-      SimpleFillSymbol polygonSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, 0xFFFFFF00, null);
+      SimpleFillSymbol polygonSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, ColorUtil.colorToArgb(Color.YELLOW), null);
       SimpleRenderer polygonRenderer = new SimpleRenderer(polygonSymbol);
       polygonGraphicOverlay.setRenderer(polygonRenderer);
       mapView.getGraphicsOverlays().add(polygonGraphicOverlay);
 
+      // polygon with curve segments (red heart) graphic
+      GraphicsOverlay curvedGraphicOverlay = new GraphicsOverlay();
+      // create a simple fill symbol with outline
+      SimpleLineSymbol curvedLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, ColorUtil.colorToArgb(Color.BLACK), 1);
+      SimpleFillSymbol curvedFillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, ColorUtil.colorToArgb(Color.RED), curvedLineSymbol);
+      SimpleRenderer curvedRenderer = new SimpleRenderer(curvedFillSymbol);
+      curvedGraphicOverlay.setRenderer(curvedRenderer);
+      // create a heart shaped graphic and add it to the map view
+      Point originPointForHeart = new Point(40e5, 5e5, SpatialReferences.getWebMercator());
+      Geometry heartGeometry = makeHeartGeometry(originPointForHeart);
+      Graphic heartGraphic = new Graphic(heartGeometry);
+      curvedGraphicOverlay.getGraphics().add(heartGraphic);
+      mapView.getGraphicsOverlays().add(curvedGraphicOverlay);
+
       // add the map view to stack pane
       stackPane.getChildren().add(mapView);
+
     } catch (Exception e) {
       e.printStackTrace();
     }
   }
 
+  /**
+   * Create a heart-shape geometry with bezier and elliptic arc segments.
+   *
+   * @param centerOfHeart the center of the square that contains the heart shape
+   * @return a heart-shape geometry
+   */
+  private Geometry makeHeartGeometry(Point centerOfHeart) {
+
+    // define the side length of the square that contains the heart shape
+    double sideLength = 10e5;
+
+    SpatialReference spatialReference = centerOfHeart.getSpatialReference();
```

Comment:
I presume the spatial reference being set here is pretty essential to the curve process? Maybe a comment above that would be useful.